components/centraldashboard/app/server.ts

```diff
@@ -19,7 +19,7 @@ import {getMetricsService} from './metrics_service_factory';
 const {
   PORT_1 = 8082,
   PROFILES_KFAM_SERVICE_HOST = 'localhost',
```

Comment:
We should address the Kfam service routing issue in this PR since we know it's possible for the centraldashboard deployment to come up before profiles does, thereby starting the server without these environment variables being set. Since the dashboard and profiles-kfam services will be deployed in the same namespace, I think it should be safe to use `profiles-kfam` as the default `PROFILES_KFAM_SERVICE_HOST` value so that the full hostname ends up being `http://profiles-kfam:8081`.

Reply:
Thus has been fixed in #3899